x-pack/plugins/observability_ai_assistant/public/components/chat/chat_item.tsx

```diff
@@ -22,9 +22,10 @@ import { ChatItemControls } from './chat_item_controls';
 import { ChatTimelineItem } from './chat_timeline';
 import { getRoleTranslation } from '../../utils/get_role_translation';
 import type { Feedback } from '../feedback_buttons';
+import type { Message } from '../../../common';
 
 export interface ChatItemProps extends ChatTimelineItem {
-  onEditSubmit: (content: string) => void;
```

Comment:
The new message can be a prompt or a function call, so a `string` didn't suffice.